src/objective-c/tests/End2EndTest/End2EndTest/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  End2EndTest
+//
+//  Created by mxyan on 7/11/16.
+//  Copyright © 2016 Google. All rights reserved.
+//
```

Comment:
I'm gonna leave them here since the file containing main function (h2_ssl.m) is complicated by itself. 

As to the license header, should we use 'Copyright 2016' instead of 2015 now? :)